VideoIndexerEnabledByArc/single-node-k8s-cluster/single-node.k8s.bicep

```diff
@@ -0,0 +1,215 @@
+@description('Azure Resource Name Prefux')
+param prefix string
+
+@description('Specifies control plane node VM size.')
+param controlPlaneNodeVmSize string = 'Standard_DS32a_v4'
+
+@description('Specifies the location.')
+param location string = resourceGroup().location
+
+@description('Specifies the address prefixes of the virtual network.')
+param virtualNetworkAddressPrefixes string = '10.0.0.0/16'
+
+@description('Specifies the address prefix of the subnet which contains the virtual machine.')
+param vmSubnetAddressPrefix string = '10.0.1.0/24'
+
+@description('Specifies the name of the network security group associated to the subnet hosting the virtual machine.')
+param vmSubnetNsgName string = 'VmSubnetNsg'
+
+var vnetName = 'Vm${prefix}vnet'
```

Comment:
shouldn't the prefix be in the beginning of the names?

Reply:
copy paste from OSS ;-)